public/views/index.html

```diff
@@ -103,16 +110,16 @@ <h4 class="modal-title" style="color: #800000">Diversity is Good, But Simplicity
         <div class="modal-body">
           <p>{{message}}</p><br>
           <p>Kindly select a region&nbsp;&nbsp;
-          <select id="selectedCountry" style="text-align:center;" ng-model="selectedCountry" >
+            <select id="selectedCountry" style="text-align:center;" ng-model="selectedCountry">
             <option ng-repeat="country in countries" value={{country.regionId}} >{{country.country}}</option>
           </select>
           </p>
         </div>
         <div class="modal-footer">
-          <button type="button" class="btn btn-primary" style="width:30%; background-color: #B22222; border:none;" data-dismiss="modal" >Close</button>
+          <button type="button" class="btn btn-primary" style="width:30%; background-color: #B22222; border:none;" data-dismiss="modal">Close</button>
```

Comment:
wow, we have inline styles in here @andela-jetanuwoma , can you please extract them to a css file @andela-hngerebara